support/@trustfractal/data-host/src/popup/services/Factory.ts

```diff
@@ -0,0 +1,106 @@
+import { ApiPromise, WsProvider } from '@polkadot/api';
+import environment from '@popup/Environment';
+import { DataHost } from '@services/DataHost';
+import { FractalAccountConnector } from '@services/FractalAccount';
+import { ProtocolService } from '@services/Protocol';
+import { ProtocolOptIn } from '@services/ProtocolOptIn';
+import { StorageService } from '@services/StorageService';
+import { AggregateMultiContext, MultiContext } from '@utils/MultiContext';
+import { ValueCache } from '@utils/ReactHooks';
+
+const types = {
+  FractalId: 'u64',
+  MerkleTree: 'Raw',
+};
```

Comment:
Maybe later: We should import these from our source of truth, /blockchain/types.json I think.